eventstore/sql/create_billable_event_components.sql

```diff
@@ -80,7 +80,7 @@ CREATE OR REPLACE FUNCTION generate_billable_event_components() RETURNS SETOF bi
 	from
 		events ev
 	left join
-		valid_pricing_plans vpp on ev.plan_guid = vpp.plan_guid
+		valid_pricing_plans vpp on ev.plan_unique_id = vpp.plan_guid
```

Comment:
`ev.plan_unique_id` does not exist as a column on the events table. This is confusing me.